mcp/mastra/mcp-server.ts

```diff
@@ -12,21 +13,44 @@ import { getNextInstructionsWorkflow, routePromptWorkflow } from './verticals/ro
 // Re-export for cross-project imports
 export { getPublicAgents };
 
-function createSimplifiedWorkflowTool(workflow: Workflow) {
+// Define a minimal workflow interface for our tool wrapper
+// This matches the subset of Mastra Workflow properties we need
+interface WorkflowLike {
```

Comment:
Instead of these awkward imitation interfaces, use type aliases instead with maybe Pick or Omit from the original types.